admin/worker/worker.go

```diff
@@ -94,6 +95,35 @@ func (w *Worker) schedule(ctx context.Context, name string, fn func(context.Cont
 	}
 }
 
+func (w *Worker) scheduleCron(ctx context.Context, name string, fn func(context.Context) error, cronExpr string) error {
```

Comment:
> For this initial autoscaler, since we still haven't solved the downtime issue during auto-scaling, I would also suggest running the autoscaler at a time we know has little traffic, e.g. at 00:00 PST.
> 
> To make that work, you might need to make a small change to the background job scheduler.

@begelundmuller Good catch! So I implemented this `scheduleCron` function to enable job scheduling by `cronjob`, which is actually a suggestion from @kaspersjo .
How would you like this?

Reply:
I like this!